tracing/tracing_test.go

```diff
@@ -0,0 +1,160 @@
+package tracing
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/distribution/distribution/v3/configuration"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
```

Comment:
This does not actually bother me. I've always had the beef with `check.v1` module and was hoping to ditch it properly and figured `testify` would find its way in. We can rework this to be consistent. Thanks for bringing this up!

Reply:
I coincidentally was looking at that before I saw this comment https://github.com/distribution/distribution/pull/3882#issuecomment-1528989027

I agree that it's not ideal to now have different frameworks, but agree with @milosgajdos that we should replace `gopkg.in/check.v1`, and from that perspective I think it's good to not introduce more uses of that legacy framework in net-new code.

I created a ticket for this work (added "help-wanted" in case someone wants to start working on it);

- https://github.com/distribution/distribution/issues/3884